README.md

```diff
@@ -82,7 +82,7 @@ import Ember from 'ember';
 var MyStateObject = Ember.Object.extend();
 
 MyStateObject.reopenClass({
-  initialState() {
+  initialState(consumerContext) {
```

Comment:
`consumerContext` kinda sucks as a name, not really sure what to call it though...